examples/apis/customer/create/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/customer"
+)
+
+func main() {
+	accessToken := "{{ACCESS_TOKEN}}"
+
+	cfg, err := config.New(accessToken)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	email := "{{EMAIL}}"
+	req := customer.Request{Email: email}
```

Comment:
A própria struct já é auto descritiva, então acredito que não precisa separar em uma variável
```suggestion
	req := customer.Request{Email: "{{EMAIL}}"}
```